docs/DEVGUIDE.md

```diff
@@ -183,10 +183,12 @@ Use Helm to create the Kubernetes deployments:
 
     export VERSION=$(git describe --tags --always --abbrev=7 --dirty)
     helm install \
-        --set "registry=${REGISTRY},version=${VERSION}" \
+        --set "registry=${REGISTRY},version=${VERSION},storageType=${STORAGE_TYPE}" \
         --namespace catalog \
         ./deploy/catalog
 
+Note that we are using an environment variable that we didn't set `STORAGE_TYPE`, with a null value it will default to `etcd`.
```

Comment:
Actually, it isn't needed at all after the change to `default`. I will remove it from the docs.